pkg/s3/s3.go

```diff
@@ -48,8 +48,16 @@ func GetConfig() *aws.Config {
 		region = "us-east-1"
 	}
 
+	accessKeyID := os.Getenv("S3_ACCESS_KEY_ID")
+	secretAccessKey := os.Getenv("S3_SECRET_ACCESS_KEY")
+
+	creds := credentials.NewStaticCredentials(accessKeyID, secretAccessKey, "")
+	if (accessKeyID == "" || secretAccessKey == "") {
+		creds = nil
+	}
+
 	s3Config := &aws.Config{
-		Credentials:      credentials.NewStaticCredentials(os.Getenv("S3_ACCESS_KEY_ID"), os.Getenv("S3_SECRET_ACCESS_KEY"), ""),
+		Credentials:      creds,
```

Comment:
FYI, these credentials are only used to communicate with the s3 object store and are not related to pulling/writing images from/to the registry

Reply:
Thanks - in what scenario is the s3 store used? It wasn't immediately apparent to me if this was something that would impact our scenario or not

Reply:
the s3 store is currently used for storing the application archives and support bundles. i still think that the changes in the PR are safe and are good to have in general.